tutorials/containers/deploying-storageos-on-kubernetes/deploying-storageos-on-kubernetes.md

```diff
@@ -0,0 +1,186 @@
+---
+id: deploying-storageos-on-kubernetes
+summary: Learn how to deploy StorageOS on Charmed Kubernetes
+categories: containers, kubernetes, csi, storageos
+tags: juju, kubernetes, beginner, container, storageos, cloud
+difficulty: 3
+status: draft
+published: 2019-09-16
+author: Peter De Sousa <peter.de.sousa@canonical.com>
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+
+---
+
+# Deploying StorageOS on Charmed Kubernetes
+
+## Overview
+Duration: 1:00
+
```

Comment:
Please add an introduction here in the same way you did for the Container Registry. Again, tell the story. Background. What is the problem? How are you trying to solve it? The better you make it, the higher the chance we'll blog about it apart from just converting it into a tutorial.